Makefile

```diff
@@ -37,9 +38,11 @@ build_tags := $(strip $(build_tags))
 ifeq ($(MAKECMDGOALS),mainnet)
    GenesisHeight=2322600
    MercuryHeight=5150000
+   NetEnv=mainnet
```

Comment:
just use the GenesisHeight and MercuryHeight to confirm the flag(main\test), don't add new flag here